src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashRepository.java

```diff
@@ -360,14 +402,41 @@ public void postFinishedComment(
 
     comment = comment.concat(additionalComment);
 
-    this.client.postPullRequestComment(pullRequestId, comment);
+    // Post the "Build Finished" comment. Failure to post it can lead to
+    // scheduling another build for the pull request unnecessarily.
+    try {
+      this.client.postPullRequestComment(pullRequestId, comment);
+    } catch (StashApiException e) {
+      logger.log(
+          Level.INFO,
```

Comment:
actually, since this will cause rebuild I think we could make it a warning

Reply:
Good point. Done.